inst/tinytest/test-app-content-handlers.R

```diff
@@ -63,4 +63,21 @@ err_msg = paste0("lexical error: invalid char in json text.\n",
 expect_equal(rs[[1]], err_msg)
 expect_equal(rs[[2]], "text/plain")
 
+
+# Test static files
+files <- c("test.R", "test.txt", "testdata.csv", "testdata.rds", "testplot.jpg",
```

Comment:
Not sure if this is the best approach (hardcoding the files), or if we should get the files somewhere else.